Readme.md

```diff
@@ -1,39 +1,47 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128532814/16.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T114539)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# GridView for Web Forms - How to calculate values on the fly in batch edit mode
 
-* **[Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))**
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - Batch Edit - How to calculate values on the fly
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t114539/)**
 <!-- run online end -->
 
+This example demonstrates how to create an unbound column (**Sum**) that changes its values based on other column values on the fly in batch edit mode.
```

Comment:
See my comment above regarding "on the fly".

Reply:
```suggestion
This example demonstrates how to create an unbound column (**Sum**) that changes its values based on other column values dynamically in batch edit mode.
```